sample/ApplicationInsightsHostingStartupSample/Startup.cs

```diff
@@ -73,13 +73,13 @@ private static void ConfigureLoggingMiddleware(IApplicationBuilder app, ILoggerF
 
         public static void Main(string[] args)
         {
+            var config = new ConfigurationBuilder()
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                .AddCommandLine(args)
+                .Build();
 
             var host = new WebHostBuilder()
-                .ConfigureAppConfiguration((hostingContext, config) =>
-                {
-                    config.AddJsonFile("appsettings.json", optional: true, reloadOnChange: true);
-                    config.AddCommandLine(args);
-                })
+                .UseConfiguration(config)
```

Comment:
How does this fix it? They look like they should be functionality equivalent. I assume to test it you just visually confirmed?